refresh2022q2 - linux : upgrade version for msgpack mongodb lzop lsyncd linux

upgrade and build package to latest stable version

- [x] msgpack : NA
- [x] mongodb : NA
- [x] lzop : NA
- [x] lsyncd : v2.2.3
- [x] linux : upgrade skipped due to version downgrade

lsyncd v2.2.3 faced build error with Lua v5.4.2

`[100%] Linking C executable lsyncd
/hab/pkgs/core/binutils/2.36.1/20210826050955/bin/ld.bfd.real: CMakeFiles/lsyncd.dir/lsyncd.c.o: in function `l_exec':
lsyncd.c:(.text+0x1629): undefined reference to `lua_objlen'
/hab/pkgs/core/binutils/2.36.1/20210826050955/bin/ld.bfd.real: lsyncd.c:(.text+0x169d): undefined reference to `lua_objlen'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/lsyncd.dir/build.make:168: lsyncd] Error 1
make[1]: *** [CMakeFiles/Makefile2:111: CMakeFiles/lsyncd.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
   lsyncd: Build time: 0m4s
   lsyncd: Exiting on error`
   
It is due to incompatibility of lsyncd v2.2.3 with Lua v5.4.x; adding lsyncd to skip list and reverting upgrade of package

--- a/lsyncd/plan.sh
+++ b/lsyncd/plan.sh
@@ -1,6 +1,6 @@
 pkg_name=lsyncd
 pkg_origin=core
-pkg_version=2.2.3
+pkg_version=2.2.1
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('GPL-2.0')
 pkg_source="https://github.com/axkibe/$pkg_name/archive/release-$pkg_version.tar.gz"

